src/models/connectionCredentials.ts

```diff
@@ -33,6 +33,7 @@ export class ConnectionCredentials implements IConnectionInfo {
     public trustServerCertificate: boolean | undefined;
```

Comment:
Cannot check in until STS bump (https://github.com/microsoft/sqltoolsservice/pull/2447)